secant/Wrappers/WrappedDeeplinkManager.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  WrappedDeeplinkManager.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 15.06.2022.
+//
+
+import Foundation
+import ComposableArchitecture
+
+struct WrappedDeeplinkManager {
```

Comment:
This seems to be a Handler more than a "manager"

Reply:
updated